godotsteam.cpp

```diff
@@ -9275,6 +9294,7 @@ void Steam::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("restartAppIfNecessary", "app_id"), &Steam::restartAppIfNecessary);
 	ClassDB::bind_method(D_METHOD("steamInit", "retrieve_stats", "app_id", "embed_callbacks"), &Steam::steamInit, DEFVAL(false), DEFVAL(0), DEFVAL(false));
 	ClassDB::bind_method(D_METHOD("steamInitEx", "retrieve_stats", "app_id", "embed_callbacks"), &Steam::steamInitEx, DEFVAL(false), DEFVAL(0), DEFVAL(false));
+	ClassDB::bind_method(D_METHOD("getSteamInitResult"), &Steam::getSteamInitResult);
```

Comment:
As weird as it is, since this isn't an actual Steam-named function is use snake case.